{{ cookiecutter.repo_name }}/{{ cookiecutter.repo_name }}/settings.py

```diff
@@ -28,10 +26,10 @@ class UpdateSettings(Settings, total=False):
 
 def load_settings() -> Settings:
     return {
-        'DATA_PATH': Path(os.getenv("DATA_PATH", Path(BASE) / '..' / 'data')),
+        'DATA_PATH': Path(os.getenv('DATA_PATH', Path().absolute().parent / 'data')),
```

Comment:
"When pathsegments is empty, the current directory is assumed" - this is not the same as directory of current file. I suggest: 
```suggestion
        'DATA_PATH': Path(os.getenv('DATA_PATH', Path(__file__).absolute().parent.parent / 'data')),
```

Reply:
yup, use file location - this will completely change when working directory is different. Until that's not a dependency - it will probaby work, but when you'll add this as a dependency to another project - it will not search in a package, but from where `python` was invoked